### ISSUE TITLE
Exception does not exist in Automattic\WooCommerce\Internal\Orders

### ISSUE THREAD

**Issue by patrick-leb** [woothemes/woocommerce#34771]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When an exception is thrown in web/app/plugins/woocommerce/src/Internal/Orders/CouponsController.php, it throws this exception:
```
Uncaught Error: Class 'Automattic\WooCommerce\Internal\Orders\Exception' not found in /var/www/html/wordpress/web/app/plugins/woocommerce/src/Internal/Orders/CouponsController.php:80
Stack trace:
#0 /var/www/html/wordpress/web/app/plugins/woocommerce/src/Internal/Orders/CouponsController.php(25): Automattic\WooCommerce\Internal\Orders\CouponsController->add_coupon_discount()
#1 /var/www/html/wordpress/web/app/plugins/woocommerce/includes/class-wc-ajax.php(1166): Automattic\WooCommerce\Internal\Orders\CouponsController->add_coupon_discount_via_ajax()
#2 /var/www/html/wordpress/web/wp/wp-includes/class-wp-hook.php(307): WC_AJAX::add_coupon_discount()
#3 /var/www/html/wordpress/web/wp/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters()
#4 /var/www/html/wordpress/web/wp/wp-includes/plugin.php(476): WP_Hook->do_action()
#5 /var/www/html/wordpress/web/wp/wp-admin/admin-ajax.php(187): do_action()
#6 {main}
```

### Expected behavior

This is because it's using a namespaced `Exception` in `Automattic\WooCommerce\Internal\Orders` which doesn't exist. They should be scoped `\Exception` instead of `Exception`.

### Actual behavior

N/A

### Steps to reproduce

Fail the application of an AJAX coupon.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by namiokuzono** [comment]
Possibly this case 5648663-zen is also affected by this reported issue.

→ For right now, asked to delete + reinstall wc.

**Comment by Konamiman** [comment]
This was fixed in https://github.com/woocommerce/woocommerce/pull/34562 (included in WooCommerce 7.0). @namiokuzono was the issue fixed by updating WooCommerce?

**Comment by Konamiman** [comment]
I'll assume that this is already fixed, if not please ping me and I'll reopen the issue.
